packages/nextjs/utils/scaffold-eth/networks.ts

```diff
@@ -93,6 +93,10 @@ export function getBlockExplorerTxLink(network: Network, txnHash: string) {
  */
 export function getBlockExplorerAddressLink(network: chains.Chain, address: string) {
   const blockExplorerBaseURL = network.blockExplorers?.default?.url;
+  if (network.id === 31337) {
+    return `http://localhost:3000/address/${address}`;
```

Comment:
We need to update it to the new URL /blockexplorer/address

Also, would `/blockexplorer/address/${address}` work?  Without the localhost:3000. There might some cases when the dev env runs in a different port.